test/server/cards/03_TWI/leaders/CaptainRexFightingForHisBrothers.spec.ts

```diff
@@ -0,0 +1,81 @@
+describe('Captain Rex, Fighting for his Brothers', function () {
+    integration(function (contextRef) {
+        beforeEach(function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    leader: 'captain-rex#fighting-for-his-brothers',
+                    groundArena: ['advanced-recon-commando'],
+                    resources: 5
+                },
+                player2: {
+                    groundArena: ['droid-commando']
+                }
+            });
+        });
+
+        describe('Captain Rex\'s leader undeployed ability', function () {
+            it('should create a Clone Tropper when a friendly unit attacked this phase', function () {
```

Comment:
nit: can just merge the `it()` and `describe()` together into a single `it()` with one name, same for the below test

Reply:
👍